app/assets/stylesheets/components/avatars.scss

```diff
@@ -61,6 +61,102 @@
       height: var(--su-10);
     }
   }
+
```

Comment:
This addition will help us use "responsive" avatars - so you can control the size of the avatar independently from breakpoints (I've used it for Notifications avatars).